control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -98,15 +98,27 @@ type InfrastructureStatus struct {
 	OauthAPIServerHost      string
 	PackageServerAPIAddress string
 	Message                 string
+	PrivateRouterHost       string
```

Comment:
In [this PR](https://github.com/openshift/hypershift/pull/1803) I tried to be consistent using the term HCPRouter which can then be exposed via an internal or external Load balancer, can we stick to that naming for consistency? So this would be I think e.g. PrivateRouterHost -> InternalHCPRouterHost

Reply:
Updated